queries.sql

```diff
@@ -0,0 +1,101 @@
+.mode column
+.header on
+
+PRAGMA foreign_keys = ON;
+
+-- Write CREATE TABLE statements for tables organization, channel, user, and message
+CREATE TABLE organization (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name VARCHAR(128)
+);
+
+CREATE TABLE channel (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name VARCHAR(128),
+    organization_id INTEGER REFERENCES organization(id)
+);
+
+CREATE TABLE user (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name VARCHAR(128),
+    channel_id INTEGER REFERENCES channel(id)
+);
+
+CREATE TABLE message (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    post_time DATETIME DEFAULT CURRENT_TIMESTAMP,
+    content VARCHAR(1024),
+    user_id INTEGER REFERENCES user(id),
+    channel_id INTEGER REFERENCES channel(id)
+);
+
+CREATE TABLE user_channel (
+    user_id INTEGER REFERENCES user(id),
+    channel_id INTEGER REFERENCES channel(id)
+);
+
+
+-- Write INSERT queries to add information to the database.
+
+-- One organization, Lambda School
+INSERT into organization (name) VALUES ("Lambda School");
+
+-- Three users, Alice, Bob, and Chris
+INSERT into user (name) VALUES ("Alice");
+INSERT into user (name) VALUES ("Bob");
```

Comment:
It is also possible to write this one in a single query by separating values with coma `,` if we insert multiple values into same table.